src/optimistic_txn_db.rs

```diff
@@ -0,0 +1,360 @@
+
+use ColumnFamily;
+use DBIterator;
+use DBRawIterator;
+use DBVector;
+use Error;
+use IteratorMode;
+use Options;
+use ReadOptions;
+use WriteOptions;
+use db::Inner;
+
+use ffi;
+use libc::{c_uchar, c_int};
+use std::collections::BTreeMap;
```

Comment:
We have a "convention" to group usings the following way:
```rust
use thirdparty::...;
...

use std::...;
...

use our_lib::...;
...
```
:smile:

Reply:
Changed